src/api.js

```diff
@@ -164,7 +175,11 @@ var schema = function(validate, options) {
         }
       }
       // If JSON was valid or validation was skipped then reply with 200 OK
-      res.status(200).json(json);
+      if (req.dryRun) {
```

Comment:
We could also interscept the other response-ending methods ( e.g. `.json` to do this check as well.